src/metabase/util/field_ref.cljc

```diff
@@ -0,0 +1,36 @@
+(ns metabase.util.field-ref
+  "Utilities for encoding and decoding field refs for URL usage."
+  (:require
+   #?(:clj [metabase.util.json :as json])
+   [clojure.string :as str]
+   [metabase.util.malli :as mu])
+  #?(:cljs (:require [goog.crypt.base64 :as base64])))
+
+(mu/defn encode-field-ref-for-url :- :string
```

Comment:
Is this used somewhere and I'm just missing it?

Reply:
Good catch, I'll remote these